.github/workflows/tests-clustermesh-upgrade.yaml

```diff
@@ -59,7 +59,7 @@ env:
   kind_version: v0.20.0
   # renovate: datasource=docker depName=quay.io/cilium/kindest-node
   k8s_version: v1.29.0-rc.1
-  cilium_cli_ci_version:
+  cilium_cli_ci_version: aefd568f74ac822510fb2813db2be8915a4e0a2b
```

Comment:
Is it for testing?

Reply:
Yes, because it requires a Cilium CLI change that has not yet been merged. That commit is intended to be dropped once that change gets released, and it is the reason for the `dont-merge/blocked` label.